### DISCUSSION TITLE
Welcome to mangosbot-bots Discussions!

### DISCUSSION THREAD

**Discussion by celguar** [celguar/mangosbot-bots#27] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wanglinb741** [comment]
要是有中国语言就更好了

**Comment by wanglinb741** [comment]
不过现在也很棒了谢谢你作者
